apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -622,7 +622,7 @@ const BookingPage = ({
                   )}
                 </>
                 {/* TODO: Change name and id ="phone" to something generic */}
-                {AttendeeInput && (
+                {AttendeeInput && !disableInput && (
```

Comment:
Don't show the disabled input field. We anyways already show the selected location above